server/.clj-kondo/imports/http-kit/http-kit/httpkit/with_channel.clj

```diff
@@ -0,0 +1,16 @@
+(ns httpkit.with-channel
```

Comment:
Curious q -- what is the story of these additions? I don't think we use http-kit

Reply:
Yeah, so basically clj-kondo needs to “import” rules from dependencies before it can apply them. I did just that (imported them) by running

```
clj-kondo --lint "$(clj -Spath)" --dependencies --copy-configs --skip-lint
```

If we add new library that uses custom rules, we’ll need to redo this.

http-kit comes from `djblue/portal`, which is in our main dependencies for some reason (if you ask me, it should be on `:dev` profile)

Reply:
SGTM! Re: djblue/portal, I think we keep it in main, because sometimes we do a remote connection to production and use it.